argobots.spec

```diff
@@ -1,22 +1,27 @@
 Name: argobots
-Version: 1.0
-Release: 1%{?dist}
+Version: 1.1a1
+Release: 1%{?dist}.g7fd1987
 Summary: Lightweight, low-level threading and tasking framework
 Group: System Environment/Libraries
 License: UChicago Argonne, LLC -- Argobots License
 Url: http://www.argobots.org/
 #Source: https://api.github.com/repos/pmodels/$(NAME)/tarball/31703b1
 Source: https://github.com/pmodels/%{name}/releases/download/v%{version}/%{name}-%{version}.tar.gz
+# patch to gather all changes from v1.1a1 tag up to ULTs stack unwinding commit
+Patch0: from_v1.1a1_upto_7fd1987.patch
 
 BuildRequires: pkgconfig
 
 # to be able to generate configure if not present
 BuildRequires: autoconf, automake, libtool
 
+# need libunwind if configuring symbolic ULTs stack dumps feature
+BuildRequires: libunwind-devel
+
 %ifarch x86_64
-%global configopts --enable-sockets --enable-verbs --enable-usnic --disable-static --enable-psm --enable-psm2
+%global configopts --enable-sockets --enable-verbs --enable-usnic --disable-static --enable-psm --enable-psm2 --enable-stack-unwind
```

Comment:
`--enable-stack-unwind` isn't implicit when `libunwind-devel` is installed?

Reply:
nope, it must be explicit.

Reply:
Pity.